OverflowError: Python int too large to convert to C long under Windows for CI tests

Last CI tests run raised the error "OverflowError: Python int too large to convert to C", only for the Windows platform.

This is the first time we get this error ([see job here](https://github.com/euroargodev/argopy/actions/runs/15892938092/job/44818812464))

Error stack:

```python
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
argopy\tests\test_fetchers_data_erddap.py:127: in assert_fetcher
    assert_all(this_fetcher, cacheable)
argopy\tests\test_fetchers_data_erddap.py:103: in assert_all
    ds = this_fetcher.to_xarray(errors='raise')
argopy\fetchers.py:684: in to_xarray
    return self.load(force=True, **kwargs).data
argopy\fetchers.py:887: in load
    self._index = self.to_index(full=False)
argopy\fetchers.py:803: in to_index
    df = self._data.argo.index
argopy\xarray.py:351: in index
    dummy_argo_uid = self._dummy_argo_uid
argopy\xarray.py:272: in _dummy_argo_uid
    self.uid(
argopy\xarray.py:332: in uid
    * np.vectorize(int)(offset * wmo_or_uid + cyc).ravel()
C:\Users\runneradmin\micromamba\envs\argopy-tests\lib\site-packages\numpy\lib\function_base.py:2372: in __call__
    return self._call_as_normal(*args, **kwargs)
C:\Users\runneradmin\micromamba\envs\argopy-tests\lib\site-packages\numpy\lib\function_base.py:2365: in _call_as_normal
    return self._vectorize_call(func=func, args=vargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <numpy.vectorize object at 0x000001E90AC90580>, func = <class 'int'>
args = (array([1.385700e+09, 1.385700e+09, 1.385700e+09, ..., 3.901242e+11,
       3.901242e+11, 3.901242e+11]),)

    def _vectorize_call(self, func, args):
        """Vectorized call to `func` over positional `args`."""
        if self.signature is not None:
            res = self._vectorize_call_with_signature(func, args)
        elif not args:
            res = func()
        else:
            ufunc, otypes = self._get_ufunc_and_otypes(func=func, args=args)
    
            # Convert args to object arrays first
            inputs = [asanyarray(a, dtype=object) for a in args]
    
            outputs = ufunc(*inputs)
    
            if ufunc.nout == 1:
>               res = asanyarray(outputs, dtype=otypes[0])
E               OverflowError: Python int too large to convert to C long

C:\Users\runneradmin\micromamba\envs\argopy-tests\lib\site-packages\numpy\lib\function_base.py:2458: OverflowError
```

--- a/argopy/xarray.py
+++ b/argopy/xarray.py
@@ -329,10 +329,10 @@ class ArgoAccessor:
             if direction is not None:
                 return (
                     encode_direction(direction)
-                    * np.vectorize(int)(offset * wmo_or_uid + cyc).ravel()
+                    * np.vectorize(np.int64)(offset * wmo_or_uid + cyc).ravel()
                 )
             else:
-                return np.vectorize(int)(offset * wmo_or_uid + cyc).ravel()
+                return np.vectorize(np.int64)(offset * wmo_or_uid + cyc).ravel()
         else:
             # DECODER
             drc = decode_direction(np.sign(wmo_or_uid))

